generators/Generators.fs

```diff
@@ -80,14 +80,19 @@ type BeerSong() =
 type BinarySearch() = 
     inherit GeneratorExercise()
 
+    let nonNegativeNumberFromNonErrorObject value =
+        match Option.ofNonErrorObject value with 
+            | None -> None
```

Comment:
Tiny nit: could you have the pipes be indented at the same level as the `match` clause?

Reply:
Changed the indentation